nova/templates/configmaps/driver.py.yaml.j2

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  driver.py: |+
+    import os
+    import types
+    import subprocess
+
+    from nova.virt.libvirt.driver import LibvirtDriver
+    from nova.compute import power_state
+
+    from oslo_log import log as logging
+    from oslo_service import loopingcall
+
+    LOG = logging.getLogger(__name__)
+    SYS_FS_CGROUP = "/sys/fs/cgroup/"
+
+
+    class K8sLibvirtDriver(LibvirtDriver):
+        def _get_compute_cgroup(self):
+            return subprocess.check_output(["bash", "-c", "cat /proc/self/cgroup | head -n 1 | awk -F ':' '{print $3}'"]).strip()
```

Comment:
```
with open("/proc/self/cgroup") as file:
    data = file.read()
    print data.split()[0].split(':')[2].strip()
```
I prefer to avoid using check_output if it is not necessary :)